ExampleCodes/FFT/Basic/GNUmakefile

```diff
@@ -0,0 +1,29 @@
+AMREX_HOME ?= ../../../../amrex
+
+DEBUG = FALSE
+DIM = 3
+COMP = gcc
+TINY_PROFILE = TRUE
+USE_MPI = TRUE
+USE_CUDA = FALSE
+USE_HIP = FALSE
+
+BL_NO_FORT = TRUE
+
+include $(AMREX_HOME)/Tools/GNUMake/Make.defs
+include $(AMREX_HOME)/Src/Base/Make.package
+include $(AMREX_HOME)/Src/FFT/Make.package
+include Make.package
+
+ifeq ($(USE_CUDA),TRUE)
+  libraries += -lcufft
+else ifeq ($(USE_HIP),TRUE)
+  # Use rocFFT.  ROC_PATH is defined in amrex
+  INCLUDE_LOCATIONS += $(ROC_PATH)/rocfft/include
+  LIBRARY_LOCATIONS += $(ROC_PATH)/rocfft/lib
+  LIBRARIES += -L$(ROC_PATH)/rocfft/lib -lrocfft
+else
+  libraries += -lfftw3_mpi -lfftw3f -lfftw3
+endif
```

Comment:
```suggestion
```
This is no longer needed because amrex's make system is handling it.

Reply:
@WeiqunZhang on my local workstation (no gpu) I still seem to need
libraries += -lfftw3_mpi -lfftw3f -lfftw3
or compile fails

Reply:
Oh, I see. Add `USE_FFT=TRUE`.

Reply:
Also the cuda compile files on gigan when removing these lines.